internal/controller/core/recipes.go

```diff
@@ -24,17 +24,15 @@ spec:
     - replicaset
     - persistentvolumeclaims
     - pods
-    includedResources:
-    - namespaces
-    - nodes
     includedNamespaces:
-    - ${ALL_NAMESPACES}
+    - ${VM_NAMESPACE}
```

Comment:
I need to check if this is required. I think the selector we build to pass to velero automatically takes the first namespace in protectedNamespaces if includedNamespaces is not provided.